codemeta.jsonld

```diff
@@ -75,6 +75,7 @@
       "readme": { "@id":"codemeta:readme", "@type": "@id" },
       "issueTracker": { "@id":"codemeta:issueTracker", "@type": "@id" },
       "referencePublication": { "@id": "codemeta:referencePublication", "@type": "@id"},
-      "maintainer": { "@id": "codemeta:maintainer" }
+      "maintainer": { "@id": "codemeta:maintainer" },
+      "hasSourceCode": { "@id": "schema:SoftwareSourceCode", "@type": "@id"},
```

Comment:
Final comma on this line is a syntax error and needs to be removed.

Also, it would be better if `hasSourceCode` were grouped with the other schema.org properties following the patterns in the file so far.

Reply:
You can see the error on the JSON-LD playground:  https://json-ld.org/playground/#startTab=tab-expanded&json-ld=https%3A%2F%2Fraw.githubusercontent.com%2Fcodemeta%2Fcodemeta%2F61f2527b69329e6bf1fae28e6cbb9340b8e554cc%2Fcodemeta.jsonld

Reply:
I changed it, there was a typo. It should be fixed now. Sorry!

Reply:
> Also, it would be better if `hasSourceCode` were grouped with the other schema.org properties following the patterns in the file so far.
@mbjones the `hasSourceCode` is not a schema.org property even if it makes sense as a schema.org property.